### ISSUE TITLE
Migrating bioinfo formulae from homebrew-science

### ISSUE THREAD

**Issue by audy** [brewsci/homebrew-bio#15]
There are quite a few bioinformatics formulae over in brewsci/homebrew-science. Can I start migrating these over? Do you have the command for migrating a formula + preserving git history?

**Comment by sjackman** [comment]
There's a corresponding issue over at https://github.com/brewsci/homebrew-science/issues/9
Look up the formula in this table: https://gist.github.com/sjackman/d1db0d73597b674bb23b51221a2914cd#file-metrics-bioinformatics-macos-tsv
If it is notable or popular (more than 50 downloads), it may be a candidate for migrating to Homebrew/core. That's your first choice. I'd focus on those formulae first.
Otherwise if it's a formula that you personally care about, you can migrate it to this repo.
The command is…
```
brew tap linuxbrew/developer
brew migrate-formula --tap=brewsci/bio brewsci/science/FOO
```
That command does not yet retain history. I don't have the time to add the history feature at this time. The history will of course be retained in the brewsci/science. If you want the history retained in brewsci/bio, you can implement that feature, or I may get around to it myself, but not until February.

**Comment by tseemann** [comment]
From my understanding, Homebrew/core only supports macOS ?
And they don't allow `if os.mac?` shims?

Does this mean any Linux patches need to be added into Linuxbrew fork of core?
And manually merged every time?

For example, `vsearch.rb` has this:
```
  depends_on "autoconf" => :build
  depends_on "automake" => :build

  unless OS.mac?
    depends_on "bzip2"
    depends_on "zlib"
  end
```

**Comment by sjackman** [comment]
Yes, Linuxbrew/core is a fork of Homebrew/core, and Homebrew/core is merged into Linuxbrew/core daily, and merged if there are merge conflicts.

**Comment by sjackman** [comment]
@audy Brewsci/bio is all up and running! Feel free to migrate any bioinformatics formulae from Brewsci/science that you like.

**Comment by sjackman** [comment]
@iml Hi, Ian! Are you interested in contributing to Brewsci/bio?

**Comment by sjackman** [comment]
@audy Hi, Austin. Are you still interesting in contributing to Brewsci/bio?

**Comment by audy** [comment]
@sjackman yes! Do you have any specific issues I could work on?

**Comment by sjackman** [comment]
Go over this list, any formulae that you personally use, and send them on over.
https://github.com/brewsci/homebrew-science/issues/9
Start with smaller, simpler formulae, and work your way up to the complicated ones.
To migrate a formula from Brewsci/science to Brewsci/bio, run…
```sh
brew tap linuxbrew/developer
brew migrate-formula --tap=brewsci/bio brewsci/science/FORMULA
```

**Comment by aersoares81** [comment]
Hi! I wanted to install Meraculous, but I thought it would be best to migrate the formula first.
First I edited the old formula because it points to an old version that's not available anymore, but now the formula breaks with the error "Error: No such file or directory @ rb_sysopen - src/c/CMakeLists.txt". Can anyone help a git newbie to fix that? I don't know why it's breaking since I just updated the URL.
